packages/by-name/kata/contrast-node-installer-image/package.nix

```diff
@@ -115,6 +119,15 @@ let
     ];
   };
 
+  kata-runtime = ociLayerTar {
```

Comment:
I think we can place this in the same layer as containerd-shim (maybe use use kata-runtime as name), as those files come from the same build and will likely be updated togehter.